Directory.Build.props

```diff
@@ -140,8 +144,9 @@
     <!-- We build linux-musl-arm on a ubuntu container, so we can't use the toolset build for alpine runtime. We need to use portable linux RID for our toolset in order to be able to use it. -->
     <ToolRuntimeRID Condition="'$(_runtimeOS)' == 'linux-musl' AND $(ArchGroup.StartsWith('arm')) AND !$(HostArch.StartsWith('arm'))">linux-x64</ToolRuntimeRID>
 
-    <!-- There are no WebAssembly tools, so treat them as Windows -->
-    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'WebAssembly'">win-x64</ToolRuntimeRID>
+    <!-- There are no WebAssembly tools, so use the default ones -->
+    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'webassembly' AND '$(OS)' == 'Windows_NT'">win-x64</ToolRuntimeRID>
+    <ToolRuntimeRID Condition="'$(RuntimeOS)' == 'webassembly' AND '$(OS)' != 'Windows_NT'">linux-x64</ToolRuntimeRID>
```

Comment:
What about mac?

Reply:
I'll add support for building wasm on mac.